sources/packages/web/src/components/layouts/aest/AESTApplicationSideBar.vue

```diff
@@ -81,7 +93,22 @@ export default defineComponent({
           });
         },
       },
+      restrictionsManagement: {
+        label: "Restrictions Management",
```

Comment:
I know the technical AC states `Restrictions Management` but I would prefer to keep this same as the title of the page which is "Application Restriction Management". 

Thoughts @andrewsignori-aot @guru-aot @andrepestana-aot @sh16011993 @katrina-aot-git @bidyashish

Reply:
I agree: `Application Restriction Management` targets restricting applications.

Reply:
Updated.

Reply:
As stated in most recent comment, Changing to `Application Restriction Management` hides some content in the side nav and hence please keep it as `Restrictions Management`